src/CheckOut.js

```diff
@@ -291,6 +303,24 @@ class CheckOut extends React.Component {
     }
   }
 
+  showBlockModal = (patron) => {
+    const {
+      resources,
+    } = this.props;
+
+    const selManualPatronBlocks = get(resources, ['manualPatronBlocks', 'records'], []);
+    let manualPatronBlocks = selManualPatronBlocks.filter(p => p.borrowing === true);
```

Comment:
@Dmitriy-Litvinenko could you use just single filter here to filter by `borrowing` and `expirationDate`?

Reply:
Fixed